mmv1/third_party/terraform/fwprovider/framework_provider.go.tmpl

```diff
@@ -319,6 +319,7 @@ func (p *FrameworkProvider) DataSources(_ context.Context) []func() datasource.D
         resourcemanager.NewGoogleClientConfigDataSource,
         resourcemanager.NewGoogleClientOpenIDUserinfoDataSource,
         {{- if ne $.TargetVersionName "ga" }}
+        resourcemanager.NewGoogleProjectFWDataSource,
```

Comment:
let's move the datasource to test-only for now, like in https://github.com/GoogleCloudPlatform/magic-modules/pull/12741/files#diff-7c30a2afcc18eb3bf49388e586c1bcb0f84fcb5768c5a9366d4d34f0dd9dd548